test/js/sql/sqlite-sql.test.ts

```diff
@@ -1311,6 +1311,130 @@ describe("SQL helpers", () => {
     expect(results[0].value).toBe("test");
   });
 
+  test("insert select helper with IN", async () => {
+    const random_name = "test_" + randomUUIDv7("hex").replaceAll("-", "");
+    await sql`CREATE TEMPORARY TABLE ${sql(random_name)} (id int, name text, age int)`;
+    {
+      await sql`INSERT INTO ${sql(random_name)} ${sql({ id: 1, name: "John", age: 30 })}`;
+      const result = await sql`SELECT * FROM ${sql(random_name)}`;
+      expect(result[0].id).toBe(1);
+      expect(result[0].name).toBe("John");
+      expect(result[0].age).toBe(30);
+    }
+    await sql`CREATE TEMPORARY TABLE ${sql(random_name + "2")} (id int, name text, age int)`;
+    {
+      await sql`INSERT INTO ${sql(random_name + "2")} (id, name, age) SELECT id, name, age FROM ${sql(random_name)} WHERE id IN ${sql([1, 2])}`;
+      const result = await sql`SELECT * FROM ${sql(random_name + "2")}`;
+      expect(result[0].id).toBe(1);
+      expect(result[0].name).toBe("John");
+      expect(result[0].age).toBe(30);
+    }
+  });
+
+  test("update helper with undefined values", async () => {
+    const random_name = "test_" + randomUUIDv7("hex").replaceAll("-", "");
```

Comment:
I've spoken my mind on random names, but if you really want to keep it, then you have a good opportunity for a closure:

```suggestion
    const testUtils = {
        newTableName: () => "test_" + randomUUIDv7("hex").replaceAll("-", ""),
    };
```

That way you can reuse it and save yourself some keys and make the code drier.

Reply:
we could just have a newTableName function in this case to remove duplicates, will do followup since it would change all tests in a lot of files